pkg/apis/bundle/v1alpha1/component_package.proto

```diff
@@ -93,6 +93,12 @@ message ComponentPackageSpec {
   repeated File raw_text_files = 6;
 }
 
+// Unstructured represents raw Kubernetes objects and corresponds to the
+// Kubernetes `unstructured.Unstructured` type from apimachinery.
+message Unstructured {
+  google.protobuf.Struct zbject = 1;
```

Comment:
zbject is a typo?

Reply:
Oops. that was for testing but I forgot to remove.